package.json

```diff
@@ -62,6 +63,7 @@
     "cross-env": "^7.0.3",
     "dayjs": "^1.11.9",
     "dumi": "^2.2.1",
+    "execa": "^5.1.1",
```

Comment:
Execa 5. * Use `require` get  pkg , but execa latest 7. * Use`import` .
 `execa@7. *` Unable to run successfully under ts-node, I tried to modify some configurations. But it has never been successful.